src/Microsoft.Owin.Security.Cookies.Shareable/CookieAuthenticationExtensions.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.DataProtection;
+using Microsoft.Framework.Internal;
+using Microsoft.Owin.Security.Cookies;
+using Microsoft.Owin.Security.Cookies.Shareable;
+
+namespace Owin
+{
+    public static class CookieAuthenticationExtensions
+    {
+        public static IAppBuilder UseCookieAuthentication(
```

Comment:
I think the full sample should live in identity repo since it pulls in a ton of stuff to actually do the full flow (EF + MVC + Identity cookie sharing).  But a tiny cookie sharing sample should be fine too